### PR TITLE
#10 Update readme

### LINKED ISSUE #10: Add usage documentation
- how to configure
- where search generated files
- how call generated classes

### PR DESCRIPTION
**Pull request by Tetraquark**
Closes #10 - readme update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,19 +67,84 @@ project build.gradle
 apply plugin: "dev.icerock.mobile.multiplatform-network-generator"
 
 dependencies {
-    commonMainApi("dev.icerock.moko:network:0.6.0")
+    commonMainApi("dev.icerock.moko:network:0.6.0")
```
Comment (Alex009) [inline README.md:70]: 0.7.0

File: README.md [hunk 2]
```diff
@@ -67,19 +67,84 @@ project build.gradle
...
+There is module **moko-network-errors** for [moko-errors](https://github.com/icerockdev/moko-errors)
+library that contains built-in mappers for mapping **moko-network** exception classes into 
+`StringDesc` objects (there are built-in resources for text in English and Russian).
+
+To use the **moko-network-errors** module just call the `registerAllNetworkMappers` extension of 
+`ExceptionMappersStorage` object:
+
+```kotlin
+ExceptionMappersStorage.registerAllNetworkMappers()
+```
```
Comment (Alex009) [inline README.md:139]: ExceptionHandler sample too

File: README.md (hunk 1 above)
Comment (Tetraquark) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (Tetraquark) [reply README.md]: Small sample was added.

**Pull request merged**